listenbrainz/webserver/static/js/src/user/Listens.tsx

```diff
@@ -687,19 +688,30 @@ export default class Listens extends React.Component<
               />
             )}
             <ListenCountCard user={user} listenCount={listenCount} />
-            {user && (
-              <div
-                className="card hidden-xs hidden-sm"
-                style={{ paddingTop: "1.5em" }}
-              >
-                <UserSocialNetwork user={user} newAlert={newAlert} />
-              </div>
-            )}
+            {user && <UserSocialNetwork user={user} newAlert={newAlert} />}
           </div>
           <div className="col-md-8 col-md-pull-4">
             {!listens.length && (
-              <div className="lead text-center">
-                <p>No listens yet</p>
+              <div className="empty-listens">
+                <FontAwesomeIcon icon={faRecordVinyl as IconProp} size="10x" />
+                <div className="lead empty-text">Empty record</div>
+                <div className="empty-message text-muted">
+                  {currentUser?.name === user?.name
+                    ? "You haven't"
+                    : `${user.name} hasn't`}{" "}
+                  listened to any songs yet.
+                </div>
+                {currentUser?.name === user?.name && (
+                  <div className="empty-action">
+                    Import <a href="/profile/import/">your listening history</a>{" "}
+                    and track your listens by{" "}
+                    <a href="/profile/music-services/details/">
+                      connecting to a music service
```

Comment:
```suggestion
                      connecting to a music streaming service
```

Reply:
The reason I didn't consider writing "_streaming_" originally was because the [music services page](https://listenbrainz.org/profile/music-services/details/) has an option to also connect with CritiqueBrainz, which is a music service but isn't a music streaming service.

Need I be this particular about the wording?

Reply:
I think in this context (getting new user onboarded) I would prefer spelling it out, even though CB is also in there.